README.md

```diff
@@ -2,6 +2,26 @@
 
 SDK to be used to access Pasqal Cloud Services.
 
+## Version Support & Deprecation Timeline
+
+| Version | Release Date | End of Support | Notes                                       |
```

Comment:
I think we can do without the notes section. People can just open the changelog if they want to look. Just the first 3 columns would be sufficient.

Reply:
Fair enough, removed